scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,67 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    function post-start-paths()
+    {
+	declare -a fnames
+	idx=0
+	for fname in $(find /var/vcap/jobs/*/bin -name post-start) ; do
+	    fnames[$idx]=$fname
+	    idx=$((idx + 1))
+	done
+	echo ${fnames[*]}
+    }
+
+    psp=$(post-start-paths)
+
+    if [ "X$psp" != X ] ; then
+	# We have post-start scripts to run.
+	# We
+	# - Start a transient monit in the background, treat it as
+	#   non-background however
+	# - Wait for all jobs (except `ready`) to be ok.
+	# - Run the post-start scripts.
+	# - Get rid of the transient monit
+	# - Mark job `ready` as ok, and start the final monit,
+	#   replacing ourselves.
+	# The last two operations are also done when we have no
+	# post-start scripts.
+
+	function monit_report_down() {
+	    # 'monit report down' requires monit 5.18+. BOSH uses monit 5.6.
+	    monit status|grep status|grep -v monitoring|grep -v 'Accessible\|Running'|wc -l
```

Comment:
Oh. `monit` is pulled in via `apt-get install ... monit ...`. (See `scripts/dockerfiles/Dockerfile-base`)
So, we get whatever version Ubuntu is giving us.
And that depends on the Ubuntu version used to assemble the docker image, right ?
   * 12.04: 5.3.2
   * 14.04: 5.6-2
   * 16.04: 5.16-2
   * 16.10: 5.19.0-1
   * zesty: 5.20.0-4

Do we want to build monit on our own, to get our specific version?
Ok, according to https://mmonit.com/wiki/Monit/Installation building is trivial.

Regarding licensing the main page (https://mmonit.com/monit/) claims

```
You can redistribute Monit and/or modify Monit under the terms
of the GNU Affero General Public License (AGPL). The license
only applies if you plan to distribute Monit to third parties.
```

Talked more in the standup ...